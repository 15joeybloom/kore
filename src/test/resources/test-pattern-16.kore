[]

module TEST-PATTERN-16

  axiom{}
    #X:#S{} []

endmodule []
