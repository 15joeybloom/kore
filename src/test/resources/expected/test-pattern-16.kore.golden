Definition
    { definitionAttributes = Attributes []
    , definitionModules =
        [ Module
            { moduleName = ModuleName "TEST-EXCEPTION-9"
            , moduleSentences =
                [ MetaSentence (SentenceAxiomSentence SentenceAxiom
                    { sentenceAxiomParameters = []
                    , sentenceAxiomPattern =
                        Fix (UnifiedPattern (UnifiedMeta (Rotate31 (VariablePattern Variable
                            { variableName = (Id "#X" AstLocationNone) :: Id Meta
                            , variableSort =
                                SortActualSort SortActual
                                    { sortActualName = (Id "#S" AstLocationNone) :: Id Meta
                                    , sortActualSorts = []
                                    }
                            }))))
                    , sentenceAxiomAttributes = Attributes []
                    })
                ]
            , moduleAttributes = Attributes []
            }
        ]
    }